airbyte-integrations/connectors/source-intercom/integration_tests/integration_test.py

```diff
@@ -3,10 +3,139 @@
 #
```

Comment:
this file was reset by reverting [this PR](https://github.com/airbytehq/airbyte/pull/23013)